packages/devtools_app/lib/src/profiler/cpu_profiler_controller.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../profiler/cpu_profile_model.dart';
+import '../profiler/cpu_profile_service.dart';
+import '../profiler/cpu_profile_transformer.dart';
+import '../ui/fake_flutter/fake_flutter.dart';
+
+class CpuProfilerController {
+  CpuProfilerController() {
+    dummyEmptyCpuProfileData = CpuProfileData.parse({});
```

Comment:
I don't see any other writes to either of these fields.  Can they be final instead?

Reply:
Done